lang/cs/Org.Apache.REEF.Network.Examples.Client/Org.Apache.REEF.Network.Examples.Client.csproj

```diff
@@ -52,6 +52,10 @@ under the License.
     <None Include="$(SolutionDir)\Org.Apache.REEF.Client\run.cmd">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="..\App.config">
+      <Link>App.config</Link>
+    </None>
+    <None Include="packages.config" />
```

Comment:
Package.config should be included in the project otherwise referenced NuGet won't be installed. We should treat each project independently assuming any project can be referenced from other places. 
If any package.config doesn't contain any NuGet but style only, I am not sure if we only open this project in VS without open others, or just build this particular project on command line, what would happen? 
For consistency, I would think we should always include package.config in projects.